src/main/java/com/walmartlabs/x12/util/split/X12TransactionSplitter.java

```diff
@@ -277,6 +316,19 @@ private void throwParserExceptionUnexpectedEnd(String actualSegmentId) {
         sb.append("unexpectedly ran out of segments - last segment id (").append(actualSegmentId).append(")");
         throw new X12ParserException(new X12ErrorDetail("", "", sb.toString()));
     }
+    
+    /**
+     * allow construction w/o rules
+     */
+    public X12TransactionSplitter() {
+    }
+    
+    /**
+     * allow construction w/ rules
+     */
+    public X12TransactionSplitter(List<X12Rule> rules) {
+        this.rules = rules;
+    }
```

Comment:
Should these go at the top of the class?

Reply:
will look at that in next PR